sadl3/com.ge.research.sadl.parent/com.ge.reasearch.sadl.errorgenerator/src/sadlMessages.properties

```diff
@@ -20,15 +20,15 @@ atpt_no_selection.description=When running ATPT, you must select the right item
 atpt_no_xml_selected=No folder or xml file selected for ATPT
 atpt_no_xml_selected.description=To run ATPT from eclipse, you must select the associated common test procedure (CTP) \
 									output file or folder which contains the CTP output file(s).
-cannot_assign_existing=Cannot assign {0}: property '{1}' already has {0} assigned to '{2}'.
+cannot_assign_existing=Cannot assign {0}: property ''{1}'' already has {0} assigned to ''{2}''.
```

Comment:
We should apply these changes on the `com.ge.research.sadl.errorgenerator` project and not on the `com.ge.reasearch.sadl.errorgenerator`. I will make those changes and make sure, the project, with the typo, is gone.

Please note, there is a typo in the latter one.